### PR TITLE
add route api test

### PR DESCRIPTION
**Pull request by songleo**
Fix https://github.com/open-cluster-management/backlog/issues/17505
Fix https://github.com/open-cluster-management/backlog/issues/17504
Signed-off-by: Song Song Li <ssli@redhat.com>

### CODE REVIEW

File: tests/pkg/tests/observability_route_test.go [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+				url,
+				nil)
+			klog.V(5).Infof("request URL: %s\n", url)
+			if err != nil {
+				return err
+			}
+
+			tr := &http.Transport{
+				/* #nosec */
+				TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```
[the remark this answers is not in the document]
Comment (songleo) [reply tests/pkg/tests/observability_route_test.go:54]: @clyang82 yes, I think we can let me add CA to try it.

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**